<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MS</string>
  </property>
  <widget class="QWidget" name="centralWidget">
<<<<<<< HEAD
   <widget class="QComboBox" name="structureComboBox">
=======
   <widget class="QComboBox" name="comboBox">
>>>>>>> d604a7cc13a2425700e397add6b027e968a30168
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>AVL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Binary Tree</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Red Black Tree</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Linked List</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Data Estructure:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Word:</string>
    </property>
   </widget>
<<<<<<< HEAD
   <widget class="QLineEdit" name="word">
=======
   <widget class="QLineEdit" name="lineEdit">
>>>>>>> d604a7cc13a2425700e397add6b027e968a30168
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Ratio:</string>
    </property>
   </widget>
<<<<<<< HEAD
   <widget class="QLineEdit" name="ratio">
=======
   <widget class="QLineEdit" name="lineEdit_2">
>>>>>>> d604a7cc13a2425700e397add6b027e968a30168
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
<<<<<<< HEAD
   <widget class="QPushButton" name="generateButton">
=======
   <widget class="QPushButton" name="pushButton">
>>>>>>> d604a7cc13a2425700e397add6b027e968a30168
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QComboBox" name="dirComboBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="selectButton">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>40</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Select file</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>341</width>
      <height>141</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Word 1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Word 2</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>374</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
